packages/commonwealth/server/routes/getSnapshotProposal.ts

```diff
@@ -0,0 +1,40 @@
+import { Response } from 'express';
+import { DB } from '../models';
+import { success, TypedRequestQuery } from '../types';
+import fetchNewSnapshotProposal from '../util/fetchSnapshot'
+
+export const Errors = {
+  NoId: 'No id was provided, cannot fetch snapshot proposal',
+};
+
+type GetSnapshotProposalReq = { id: string };
+
+const getSnapshotProposal = async (
+  models: DB,
+  req: TypedRequestQuery<GetSnapshotProposalReq>,
+  res: Response,
```

Comment:
Mixing old and new route code?

Reply:
@rvmelkonian not sure where this route was meant to be used